crates/cargo-winrt/src/main.rs

```diff
@@ -332,7 +332,7 @@ impl DependencyDescriptor {
     }
 }
 
-fn try_download(url: String, recursion_amount: u8) -> anyhow::Result<Vec<u8>> {
+fn try_download(url: String, recursion_amount: u8, redirect: bool) -> anyhow::Result<Vec<u8>> {
```

Comment:
The formal parameter name (`redirect`) doesn't quite match the argument's meaning. Something like `request_body` or `include_body` is maybe a better fit.

Reply:
Changed to request_nobody.

Reply:
I don't not think this is not an improvement ;) For one, it exposes naming details of the currently used library. If that library ever gets swapped out for another library with different naming (say `no_body()` rather than `nobody()`), the parameter name no longer matches.

What's more troublesome, as I hinted to above: Using a negative in a parameter name is an invitation for double and triple negations in code, with an unavoidable tendency to make it harder to comprehend. Plus, the name suggests that you are *"requesting no body"*, when really the semantics are to *"not request a body"*.

Though I have no say in this. I'm not affiliated with this repository, just a user of this library. This is really only a comment.

Reply:
As @tim-weis mentioned, I'm also not super keen on this negation in the parameter name. Maybe more than that for me, I have a real brain problem parsing `request_nobody` and even in that case would definitely prefer `request_no_body`, but maybe `nobody` is just a common term of art and I'm not steeped enough in HTTP culture to know that.

I would definitely prefer `include_body` in this case for both of those reasons, but I'm also just a humble drive-by commenter. :)

Reply:
`include_body` seems the cleanest without imposing too much rework on a simple internal function. If this were a library I'd avoid a bool parameter altogether.

Reply:
Thanks for comments, changed to include_body